### ISSUE TITLE
How to change the loss function?

### ISSUE THREAD

**Issue by jodumagpi** [open-mmlab/mmdetection#2473]
I tried training on my own dataset and it was successful.

However, I want to do some tweaks in the loss function so I tried to do some changes on some of the python codes under **mmdetection/mmdet/models/losses/** but there seems to be no change at all. I even tried to set the return value to zero but the training indicates that there is still a mask loss.

What is the proper way of defining our own loss function?

**Comment by jodumagpi** [comment]
I'm sorry. This was a duplicate of the previous question I had.
